docs/usage/self-hosted-configuration.md

```diff
@@ -755,6 +755,33 @@ If left as default (null), a random short ID will be selected.
 
 ## logFileLevel
 
+## mergeConfidenceEndpoint
+
+If set, Renovate will query this API for Merge Confidence data.
+
+If you use the Mend Renovate Enterprise Edition (Renovate EE) and:
+
+- have a static merge confidence token that you set via `MEND_RNV_MC_TOKEN`
+- _or_ set `MEND_RNV_MC_TOKEN` to `auto`
+
+Then you must set this variable at the _server_ and the _workers_.
```

Comment:
Do we need to explain _how_ and in which config file(s) to set the variable at the server and the workers?

Reply:
no, it's not usable for the free oss renovate, you get docs when you pay for that mend feature

Reply:
What should we say then?

- Read the docs you get from Mend to learn how to set the variable.
- Read the docs at _insert link to docs here_.
- Something else?
- Leave blank.